apps/web/src/pages/TalentNominations/NominationGroup/CareerExperience.tsx

```diff
@@ -1,40 +1,185 @@
+          <Heading
+            Icon={FlagIcon}
+            level="h2"
+            color="primary"
+            data-h2-margin="base(x1.5 x1.5 0 x1.5)"
+            data-h2-font-weight="base(400)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Current position",
+              id: "elMgS2",
+              description: "Heading for current position",
+            })}
+          </Heading>
+          <p data-h2-margin="base(x.5 x1.5 x1 x1.5)">
+            {intl.formatMessage({
+              defaultMessage:
+                "This section shows the candidate's current role. If it's an acting role, the candidate's substantive role will also appear here if they've provided it. Select individual experiences to see more details.",
+              id: "R1QSIY",
+              description:
+                "Description for the career page current role section",
+            })}
+          </p>
+          {shareProfile && (
+            <Separator data-h2-margin="base(0 0 x1 0)" space="none" />
+          )}
+          {shareProfile && (
+            <div data-h2-margin="base(0 x1.5)">
+              <CurrentPositionExperiences query={workExperiences} />
+            </div>
+          )}
+          {!shareProfile && (
+            <Well data-h2-margin="base(0 x1.5 x1.75 x1.5)" color="error">
+              <p
+                data-h2-margin-bottom="base(x1)"
+                data-h2-font-weight="base(700)"
+              >
+                {intl.formatMessage({
+                  defaultMessage:
+                    "This nominee has not agreed to share their information with your community",
+                  id: "4ujr5X",
+                  description: "Null message for nominee profile",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Nominees can agree to provide access to their profile using the “Functional communities” tool on their dashboard.",
+                  id: "8plD42",
+                  description: "Null secondary message for nominee profile",
+                })}
+              </p>
+            </Well>
+          )}
+          {shareProfile && (
+            <Separator data-h2-margin="base(x1 0)" space="none" />
+          )}
+          {shareProfile && (
+            <p
+              data-h2-color="base(black.light)"
+              data-h2-margin="base(0 x1.5 x1.75 x1.5)"
+            >
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    "Nominee's profile was last updated on: {lastUpdated}",
+                  id: "guYbEb",
+                  description: "Last time user's profile was updated",
+                },
+                { lastUpdated },
+              )}
+            </p>
+          )}
+        </div>
+      </CardBasic>
+      {/* <CardBasic>
+        Full career card
+      </CardBasic> */}
+    </Pending>
+  );
```

Comment:
I think this is where the Full career card can be plugged in

Reply:
yeah .. this looks much cleaner .. I'm going to base off this